pillarbox-core-business/src/test/java/ch/srgssr/pillarbox/core/business/tracker/commandersact/CommandersActTrackerIntegrationTest.kt

```diff
@@ -351,33 +361,40 @@ class CommandersActTrackerIntegrationTest {
 
         assertEquals(3, tcMediaEvents.size)
 
-        assertEquals(Play, tcMediaEvents[0].eventType)
-        assertTrue(tcMediaEvents[0].assets.isNotEmpty())
-        assertNull(tcMediaEvents[0].sourceId)
-
-        assertEquals(Pause, tcMediaEvents[1].eventType)
-        assertTrue(tcMediaEvents[1].assets.isNotEmpty())
-        assertNull(tcMediaEvents[1].sourceId)
-
-        assertEquals(Play, tcMediaEvents[2].eventType)
-        assertTrue(tcMediaEvents[2].assets.isNotEmpty())
-        assertNull(tcMediaEvents[2].sourceId)
+        tcMediaEvents[0].let {
+            assertEquals(Play, it.eventType)
+            assertTrue(it.assets.isNotEmpty())
+            assertNull(it.sourceId)
+        }
+        tcMediaEvents[1].let {
+            assertEquals(Pause, it.eventType)
+            assertTrue(it.assets.isNotEmpty())
+            assertNull(it.sourceId)
+        }
+        tcMediaEvents[2].let {
+            assertEquals(Play, it.eventType)
+            assertTrue(it.assets.isNotEmpty())
+            assertNull(it.sourceId)
+        }
     }
 
     @Test
     fun `player prepared, playing and stopped`() {
         val tcMediaEvents = mutableListOf<TCMediaEvent>()
 
-        player.setMediaItem(SRGMediaItemBuilder(URN_LIVE_VIDEO).build())
+        player.setMediaItem(SRGMediaItemBuilder(URN_NOT_LIVE_VIDEO).build())
         player.prepare()
         player.playWhenReady = true
 
         TestPlayerRunHelper.runUntilPlaybackState(player, Player.STATE_READY)
         TestPlayerRunHelper.runUntilPendingCommandsAreFullyHandled(player)
 
-        clock.advanceTime(2.minutes.inWholeMilliseconds)
-        player.stop()
+        val position = 2.minutes
+        TestPillarboxRunHelper.runAfterPositionReached(player, position = position, clock = clock) {
+            player.stop()
+        }
```

Comment:
You can keep it like that, I'm just curious about why this new method, instead of calling directly the following:
```suggestion
        TestPillarboxRunHelper.runUntilPosition(player, position, clock)
        player.stop()
```

Reply:
Good point, I removed it